spring-batch-core/src/main/java/org/springframework/batch/core/job/AbstractJob.java

```diff
@@ -353,6 +362,13 @@ public final void execute(JobExecution execution) {
 					execution.setExitStatus(exitStatus.and(newExitStatus));
 				}
 
+				timerSample.stop(Timer.builder(METRICS_PREFIX + "job")
+						.description("Job duration in seconds")
```

Comment:
Don't refer to "seconds" here. The actual base unit of time will depend on the monitoring system, e.g. seconds for prometheus, nanos for Atlas, millis for Datadog. Just call it "Job duration".

Reply:
Make sense. Removed the unit from all descriptions.